crud/insert.lua

```diff
@@ -58,6 +58,8 @@ local function call_insert_on_router(space_name, tuple, opts)
         return nil, InsertError:new("Space %q doesn't exist", space_name), true
     end
 
+    local tuple = table.deepcopy(original_tuple)
```

Comment:
Raw thoughts on possible optimizations:

- Skip copying on `crud.insert_object()`.
- Pass the bucket_id separately to storage and insert it into the tuple on the storage (take care to router/storage rolling code update).

I don't want to change it within this pull request, but rather to think around later.

Reply:
Second thought seems useful, thank you! I'll file a ticket for the future

Reply:
Refer to #274